src/tests/CountWordsTest.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import fileAnalysis.CountWords;
+
+import org.junit.jupiter.api.Test;
+
+class CountWordsTest {
+
+	@Test
+	public void testCountWords_Empty() {
+		assertEquals(0, CountWords.countWords(""));
+	}

```

Comment:
@Parsa-Rajabi @Lightbrite88 
Hey Everyone, 
This is a good point to go over how we should be writing our unit tests. We want to practice making our tests as readable as possible. To do this we need to:
1. Give test a very obvious function name (ex. countWordsShouldReturnZeroWhenTextIsEmpty())
2. We want to follow the general unit test structure of expected given, when, then. This will make your test significantly more readable. For this test it would look something like this:
```
@Test 
	public void countWordsShouldReturnZeroWhenTextIsEmpty() {
		// expected result
		int expectedResult = 0;

		// given
		String emptyText = "";

		// when
		int wordCount = CountWords.countWords(emptyText);

		// then
		assertEquals(wordCount, expectedResult);
	}
```
Can everyone be sure to follow this format when writing your unit tests

Reply:
sure thing